actions.yaml

```diff
@@ -41,3 +41,11 @@ create-backup:
 
 list-backups:
   description: List available backup_ids in the S3 bucket and path provided by the S3 integrator charm.
+
+restore:
```

Comment:
@delgod should we name action as `restore-backup` to be similar to `create-backup` and `list-backups`?

Reply:
Topic closed, it is expected naming (c) Mohamed:

```
juju run-action --wait mysql-k8s/0 create-backup
juju run-action --wait mysql-k8s/0 list-backups
juju run-action --wait mysql-k8s/0 restore backup-id=<%Y-%m-%d-%H-%M>
```

Reply:
Hi @taurus-forever and @delgod :
The above is what we have agreed on already . So I am fine with it :)